SECURITY.md

```diff
@@ -0,0 +1,76 @@
+
+---
+
+## Reporting a Vulnerability
+
+* **Private first.**  
+   Do **not** open a public GitHub issue or pull request.
+
+* **Use the offical Github documentation on how to report a vulnerability.**
+   [Creating a repository security advisory](https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/creating-a-repository-security-advisory#creating-a-security-advisory)
+
+### What to Expect
+
+Once you've reported a vulnerability, here's what you can expect from our security response process. We follow OpenSSF guidelines for response timing:
+
+| Action | Target time | Notes |
+| ------ | ----------- | ----- |
+| Initial maintainer response | **≤ 14 calendar days** | Acknowledge receipt and begin triage. |
+| Triage & severity scoring | **≤ 30 days** | We follow CVSS v3.1. |
+| Fix availability | **Next scheduled monthly release**<br/>(or an out-of-band patch release for Critical/High issues) | We may cut `vX.Y.Z` if waiting a full month poses undue risk. |
+
+---
+
+## Supported Versions
+
+| Release | First published | Security-fix window |
+|---------|-----------------|---------------------|
+| **Latest monthly release** | see `git tag --sort=-creatordate \| head -n 1` | Actively maintained |
+| Any prior release | — | **Unsupported** – please upgrade |
+
+> **Why no backports?**  
+> Kata’s architecture evolves quickly; back-porting patches would re-introduce the very maintenance burden we avoid by using a rolling model.
+
+---
+
+## Disclosure Process & Fix Delivery
+
+1. We develop the fix on a private branch.  
+2. Once validated, we coordinate embargo dates with downstream consumers when appropriate.  
+3. We request a CVE ID from MITRE (or another CNA) if one hasn't already been assigned.
+4. The fix and vulnerability details are published together in **either**:
+   * Common: The next regular monthly release (e.g., `v3.19`) when impact is moderate and waiting does not materially increase risk, **or**
+   * Exception: A point release (e.g., `v3.18.1`) if the vulnerability affects only the current series.
+
+---
+
+## Security Advisories & Release Notes
+
+* Each patch or monthly release includes a **Security Bulletin** section in its GitHub *Release Notes* summarizing:
+  * affected components & versions,
+  * CVE identifiers (if assigned),
+  * severity / CVSS score,
+  * mitigation steps,
+  * upgrade instructions.
+
+* We do **not** publish separate “stable-branch” advisories because unsupported branches receive no fixes.
+
+---
+
+## Frequently Asked Questions
+
+**Q: I run `v3.16` – will you patch it?**  
+A: No. Upgrade to the latest monthly release.
+
+**Q: Can I get early access to embargoed fixes?**  
+A: Only project members under the disclosure agreement (see [SECURITY_CONTACTS](SECURITY_CONTACTS)) receive advance patches.
```

Comment:
Scenario: I'm a vendor, shipping a modified downstream version of Kata with my product. Most of the time, it would not be possible for me to rebase onto the next release or backport a security fix on short notice. Even if I'm notified in advance about an upcoming, _undisclosed_ security fix, I'd still need to integrate and validate it into my downstream branch after it was published.

According to this rule, my product is vulnerable for an extended period of time, and I need to scramble to adopt the patch before its exploited. Is that intended? Could we instead have a process for advance notifications, like https://github.com/kubernetes/committee-security-response/blob/main/private-distributors-list.md?

@kata-containers/architecture-committee

Reply:
That is a good point. According to this link there is a special mailing list with ditributors that can have access to early fixes that cannot realy on a public release.